### PR TITLE
Add Exercise05

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise05/Extension.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise05
+{
+    public static class Extension
+    {
+        public static IEnumerable<List<T>> SubList<T>(this IList<T> list, int max)
+        {
+            var cnt = 0;
```
Comment (voquanghoa) [inline Exercise05/Extension.cs]: Try to simplify this 🔢

File: Exercise05/Extension.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            var newList = new List<T>();
+
+            foreach (var i in list)
+            {
+                newList.Add(i);
+                cnt++;
+                if (cnt == max || i.Equals(list.Last())) 
+                {
+                    yield return newList;
+                    cnt = 0;
```
Comment (voquanghoa) [inline Exercise05/Extension.cs]: Bad name, what is `cnt`?

File: Exercise05/Program.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace Exercise05
+{
+    class Program
+    {
+        
+
+        static void Main(string[] args)
+        {
+            var list = new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9 }.SubList(4);
```
Comment (voquanghoa) [inline Exercise05/Program.cs]: Bad name `list`

File: Exercise05/Program.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+            foreach(var i in list)
```
Comment (voquanghoa) [inline Exercise05/Program.cs]:
```
list.ForEach(x=>{
      x.ForEach(Console.WriteLine);
     Console.WriteLine();
}
)
```

Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

File: Exercise05/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

File: Exercise05/Extension.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise05/Extension.cs]: cnt is count, Done

File: Exercise05/Extension.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise05/Extension.cs]: i don't know your idea

File: Exercise05/Extension.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+namespace Exercise05
+{
+    public static class Extension
+    {
+        public static IEnumerable<List<T>> SubList<T>(this IList<T> list, int max)
+        {
+            var count = 0;
+            var newList = new List<T>();
+
+            foreach (var i in list)
```
Comment (voquanghoa) [inline Exercise05/Extension.cs]: Doesn't work!!!!!!

Comment (huynhngocminh2511) [reply Exercise05/Extension.cs]: Done

**Pull request merged**